pkg/dcl/metadata/metadata.go

```diff
@@ -232,6 +232,11 @@ var serviceList = []ServiceMetadata{
 				// This resource is implemented through Terraform/KCC bridge, only load its DCL OpenAPI schema for resource references
 				Releasable: false,
 			},
+			{
+				Kind:       "ComputeNetworkAttachment",
+				DCLType:    "NetworkAttachment",
+				Releasable: false,
+			},
```

Comment:
This missing entry was causing the CRD generation before.

Reply:
@justinsb this missing entry was actually causing the CRD generation failure. AFAIU now that failure was to be expected 😅 . I can add this gotcha to our docs for upgrading the lib if that helps.

Also note, this does not upgrade us to latest DCL lib, which last I checked was 1.63 which fails as w a deepcopy issue. I filed #1427 to track that